File: C2_-_Input_processing_and_output/E3_-_print_quotes/CPP/main.cpp

#include <iostream>

int main( void )
{
    std::string quote;
    std::string name;

    std::cout << "What is the quote? : ";
    std::getline( std::cin, quote );
    std::cout << "Who said that? : ";
    std::getline( std::cin, name );

    std::cout << name << " said : \"" << quote << "\"." << std::endl;

    return 0;
}

File: C2_-_Input_processing_and_output/E3_-_print_quotes/RUST/main.rs
use std::io::stdin;

fn main()
{
    let mut quote = String::new();
    let mut name = String::new();

    println!( "What is the quote? :" );
    let mut _ret = stdin().read_line( &mut quote );

    println!( "Who said that? :" );
    _ret = stdin().read_line( &mut name );

    println!("{} said: \"{}\".", name.trim_end(), quote.trim_end() );
}